psutil/_pslinux.py

```diff
@@ -159,8 +159,14 @@ class IOPriority(enum.IntEnum):
                 'read_time', 'write_time',
                 'read_merged_count', 'write_merged_count',
                 'busy_time'])
-popenfile = namedtuple(
-    'popenfile', ['path', 'fd', 'position', 'mode', 'flags'])
+
+popenfile = namedtuple('popenfile', ['path', 'fd', 'position', 'mode', 
+    'flags', 'mnt_id', 'anon_fields'])
+
+ppoll_tfd = namedtuple('ppoll_tfd', ['events', 'data'])
+pinotify = namedtuple('pinotify_attrs', ['ino', 'sdev', 'mask', 'ignored_mask', 
+     'fhandle_bytes', 'fhandle_type', 'f_handle'])
+
```

Comment:
Two new `namedtuple`s for `ppoll` and `inotify` descriptors.  Using the fd's as the keys and these `namedtuple`s as the values.  The other anon inodes `signalfd` and `eventfd`, don't contain multiple fields and can use just their event masks as the values.